packages/cactus-plugin-ledger-connector-iroha2/src/test/typescript/integration/iroha2-generate-and-send-signed-transaction.test.ts

```diff
@@ -613,4 +614,27 @@ describe("Generate and send signed transaction tests", () => {
     log.info("Bob (source) balance after final transfer:", finalBobBalance);
     expect(finalBobBalance).toEqual(transferValue - transferBackValue);
   });
+
+  test("get prometheus exporter metrics", async () => {
+
+    const res = await env.apiClient.getPrometheusMetricsV1();
+    const promMetricsOutput =
+      "# HELP " +
+      K_CACTUS_IROHA2_TOTAL_TX_COUNT +
+      " Total transactions executed\n" +
+      "# TYPE " +
+      K_CACTUS_IROHA2_TOTAL_TX_COUNT +
+      " gauge\n" +
+      K_CACTUS_IROHA2_TOTAL_TX_COUNT +
+      '{type="' +
+      K_CACTUS_IROHA2_TOTAL_TX_COUNT +
+      '"} 8';
+  
+
+    expect(res).toBeTruthy();
+    expect(res.data).toBeTruthy();
+    expect(res.status).toEqual(200);
+    expect(res.data.includes(promMetricsOutput)).toBeTrue();
+  });
+
```

Comment:
I think there's no need to duplicate this test, the one defined in [iroha2-setup-and-basic-operations.test.ts](https://github.com/hyperledger/cacti/pull/3461/files#diff-c28c78bb81f10102eec1a16a57cdab41bf9d8261fda6f6f94731a5cfaa6c84ef) should be enough

Reply:
Done.